forge-gui/res/cardsfolder/upcoming/experimental_lab_staff_room.txt

```diff
@@ -0,0 +1,21 @@
+Name:Experimental Lab
+ManaCost:3 G
+Types:Enchantment Room
+T:Mode$ UnlockDoor | ValidPlayer$ You | ValidCard$ Card.Self | ThisDoor$ True | Execute$ TrigManifest | TriggerDescription$ When you unlock this door, manifest dread, then put two +1/+1 counters and a trample counter on that creature.
+SVar:TrigManifest:DB$ ManifestDread | Amount$ 1 | RememberManifested$ True | SubAbility$ DBPutP1P1Counter
+SVar:DBPutP1P1Counter:DB$ PutCounter | Defined$ RememberedCard | CounterType$ P1P1 | CounterNum$ 2 | SubAbility$ DBPutTrampleCounter
+SVar:DBPutTrampleCounter:DB$ PutCounter | Defined$ RememberedCard | CounterNum$ 1 | CounterType$ Trample | SubAbility$ DBCleanup

```

Comment:
use PutCounterAll instead like on _Guide of Souls_  because this "put" should be a single event

Reply:
I don't think that effect supports Defined$ though

Reply:
You're right. It's ValidCards$

Reply:
`ValidCards$ Card.IsRemembered`
Defined (AbilityUtils) vs Valid (CardProperty) syntax